tic-tac-toe/src/Square.jsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+const Square = (props) => {
```

Comment:
One small thing - you can use object destructuring if you like to avoid having to repeat `props.` on the next line. `{clickEvent, value}`